src/components/progress-bar/progress-bar.html

```diff
@@ -0,0 +1,4 @@
+<div class="background"></div>
```

Comment:
Could you add a comment explaining why `background` doesn't _contain_ the other elements?